src/fem/fe_rt.hh

```diff
@@ -63,18 +63,28 @@ public:
     arma::vec::fixed<dim> basis_grad(const unsigned int i, const arma::vec::fixed<dim> &p) const;
 
     /**
-     * @brief Returns the @p ith basis function evaluated at the point @p p.
-     * @param i Number of the basis function.
-     * @param p Point of evaluation.
+     * @brief Calculates the value of the @p comp-th component of
+     * the @p i-th raw basis function at the
+     * point @p p on the reference element (for vector-valued FE).
+     *
+     * @param i    Number of the basis function.
+     * @param p    Point of evaluation.
+     * @param comp Number of vector component.
```

Comment:
Convenction for numbering components should be described, in FiniteElement class description.

Reply:
Numbering of components is clear, what is not intuitive, is the ordering of basis functions (differs from ordering of dofs in dof handler).